src/angularjs/src/app/components/filters/analysis-job-filter.html

```diff
@@ -2,23 +2,29 @@
     <th>ID</th>
     <th>
         <div class="dropdown">
-        <input name="neighborhood-filter" ng-model="ctl.statusFilter" title="Filter by Status"
-            placeholder="Status" type="text" class="form-control"
-            uib-typeahead="status for status in ctl.statuses |
-                           filter: $viewValue |
-                           limitTo: 8">
+        <input name="neighborhood-filter"
+               title="Filter by Status"
+               placeholder="Status"
+               ng-model="ctl.statusFilter"
+               ng-model-options="{ debounce: 400 }"
+               ng-change="ctl.onStatusFilterChanged()"
```

Comment:
Switching to `ng-change` and `typeahead-on-select` allows us to explicitly control when we set the `filters` property in the controller. This fixes a nasty side effect where the old implementation would trigger a call to `/api/analysis_jobs` in the parent controller every time a character was typed in either filter field. Now the call only explicitly happens when a meaningful change occurs.